extensions/wikidata/src/org/openrefine/wikidata/qa/WikidataConstraintFetcher.java

```diff
@@ -337,4 +342,57 @@ public boolean usableOnItems(PropertyIdValue pid) {
         }
         return results;
     }
+
+    /**
+     * Returns the list of PropertyIdValues of conflicting statements
+     *
+     * @param pid:
+     *            the property having conflicts-with constraint
+     * @return
+     */
+    public List<PropertyIdValue> getConflictsWithProperties(PropertyIdValue pid) {
+        List<Statement> statementList = getConstraintsByType(pid, CONFLICTS_WITH_CONSTRAINT_QID).collect(Collectors.toList());
+        List<Value> conflictsWithProperties = new ArrayList<>();
+        for(Statement statement : statementList){
+            List<SnakGroup> specs = statement.getClaim().getQualifiers();
+            if (specs != null) {
+                conflictsWithProperties.addAll(findValues(specs, CONFLICTS_WITH_PROPERTY_PID));
+            }
+        }
+
+        if (!conflictsWithProperties.isEmpty()) {
+            return conflictsWithProperties.stream().map(e -> e == null ? null : (PropertyIdValue) e).collect(Collectors.toList());
+        }
+        return null;
+    }
+
+    /**
+     * Returns the list of all the conflicting values
+     *
+     * @param pid:
+     *            the property having conflicts-with constraint
+     * @return
+     */
+    public List<Value> getItemWithConflicts(PropertyIdValue pid, PropertyIdValue conflictingPid) {
+        List<Statement> statementList = getConstraintsByType(pid, CONFLICTS_WITH_CONSTRAINT_QID).collect(Collectors.toList());
+        List<Value> itemList = new ArrayList<>();
+        for (Statement statement : statementList) {
+            List<SnakGroup> specs = statement.getClaim().getQualifiers();
+            boolean flag = false;
+            for(SnakGroup group : specs) {
+                for (Snak snak : group.getSnaks()) {
+                    if (snak.getValue() instanceof PropertyIdValueImpl && snak.getValue().equals(conflictingPid)){
+                        flag = true;
+                    } else if (snak.getValue() instanceof PropertyIdValueImpl && !snak.getValue().equals(conflictingPid)) {
```

Comment:
Same here.

Reply:
Here, instead of relying on the `PropertyIdValue` type of the snak value, you should rather rely on the `snak.getProperty()`'s value.